### PR TITLE
Add Contacts Page

### PR DESCRIPTION
**Pull request by ribeiropdiogo**
Created the contacts page for the contacts provided in the guidlines file. As the guidelines were a bit confusing, I implemented some small corrections in order to make everything more visually pleasant.

### CODE REVIEW

**Comment by machadovilaca** [comment]
I don't really like the design, check [jmv/update_contacts_design](https://github.com/cesium/HeartBits2020/tree/jmv/update_contacts_design) and give me your opinion.

contacts

@ribeiropdiogo @miguelbrandao

**Comment by ribeiropdiogo** [comment]
@machadovilaca i think it's very nice! Do you think the color of the icons should be the same as in the other pages or do you want to keep those colors?

**Comment by machadovilaca** [comment]
I tried the other color in all icons and I did not like it, it was too monotonous.
If you like it, then you can maybe rebase that branch into this one, I created it from this.

**Comment by ribeiropdiogo** [comment]
ok perfect then!

**Comment by machadovilaca** [comment]
In the mobile view, instead of fitting the text content to the width, can you adjust it to be one contact per line like:

```
Redes Sociais

F_logo HackathonHeartbits
I_logo @hackathon_heartbits

Organização

M_logo cesium...
M_logo com...
M_logo img_
```

**Comment by miguelbrandao** [comment]
I like the design! Just follow @machadovilaca 's suggestion and it's ready to be merged imo

**Comment by ribeiropdiogo** [comment]
@machadovilaca @miguelbrandao I tweaked the options to the ratio that seemed to look the best to me. Give me your thoughts

**Comment by machadovilaca** [comment]
To me, it feels like the contact text it's still being unnecessarily splitted. 
 Can you try to put the logo and the text in different divs? Something like 

```
<div class="contact icons-space">
  <a target="_blank" rel="noopener noreferrer" href="mailto:cesium@di.uminho.pt">
    <i class="fas fa-envelope cesium fa-5x" aria-hidden="true"></i>
  </a>
</div>

<div class="contact icons-space">
  <a target="_blank" rel="noopener noreferrer" href="mailto:cesium@di.uminho.pt">
    <h6>cesium@di.uminho.pt</h6>
  </a>
</div>
```

In the logo you could keep ` max-width: 25%; min-width: 20%` but increase those values for the text

**Comment by ribeiropdiogo** [comment]
Theres is no need to add divs. I was forcing the text to break. I removed that option and gave the contact more width in the mobile version.

**Comment by machadovilaca** [comment]
Perfect, I like it :heart:

File: _includes/contacts/contacts.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="contact icons-space">
+          <a target="_blank" rel="noopener noreferrer" href="mailto:comunicacao@anem.pt">
+            <i class="fas fa-envelope anem fa-5x" aria-hidden="true"></i>
+            <h6>comunicacao@anem.pt</h6>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
```
Comment (machadovilaca) [inline _includes/contacts/contacts.html]: Empty line at the end of file here

File: _includes/contacts/contactheader.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<section id="contactheader">
+  <div class="container">
+    <h1>Contactos</h1>
+  </div>
+</section>
```
Comment (machadovilaca) [inline _includes/contacts/contactheader.html]: Empty line at the end of file here

**Review by machadovilaca — CHANGES_REQUESTED** [review]

**Review by machadovilaca — APPROVED** [review]

**Pull request merged**